core/themes/seven/css/seven.fonts.css

```diff
@@ -14,12 +14,12 @@
   font-family: 'Open Sans';
   font-style: italic;
   font-weight: 400;
-  src: url('../fonts/OpenSans-Light-webfont.eot'); /* Old IE */
+  src: url('../fonts/OpenSans-Italic-webfont.eot'); /* Old IE */
```

Comment:
Can't believe I goofed that up! But we should put the edits to this file in a separate PR since it's off the topic here.